lib/vmdb/config/activator.rb

```diff
@@ -22,6 +22,10 @@ def log(data)
         Vmdb::Loggers.apply_config(data)
       end
 
+      def ntp(_data)
+        MiqServer.my_server.ntp_reload_queue unless MiqServer.my_server.nil? rescue nil
```

Comment:
What about `MiqServer.my_server.try(:ntp_reload_queue)` instead of all this complexity.